homeassistant/components/binary_sensor/alarmdecoder.py

```diff
@@ -122,3 +131,13 @@ def _rfx_message_callback(self, message):
         if self._rfid and message and message.serial_number == self._rfid:
             self._rfstate = message.value
             self.schedule_update_ha_state()
+
+    def _rel_message_callback(self, message):
+        """Update relay state."""
+        if (self._relay_addr == message.address and
+                self._relay_chan == message.channel):
+            _LOGGER.debug("Relay " + str(message.address) + "-" +
```

Comment:
Please pass the arguments to the message format string using the old style string formatting operator. Don't concatenate the the strings.

https://docs.python.org/3/library/logging.html#logging.Logger.debug